packages/activemq/data_stream/broker/manifest.yml

```
@@ -0,0 +1,15 @@
+type: metrics
+title: ActiveMQ broker metrics
+release: experimental
```

Experimental should be used only for features that we don't plan to give any kind of support in the long term in their current form.
```suggestion
release: beta
```

---

Done